### ISSUE TITLE
Doesn't dial with name from address book  

### ISSUE THREAD

**Issue by PhilipBanse** [Studio-Link/webapp#39]
Hi,

When I just put the name ("Peter" oder "Peter Bauer") from an address book entry ("Peter Bauer") in the dial field, it throws an 404 and terminates the dialing. When I enter the SIP ID it works.

**Comment by sreimers** [comment]
Yes the address book is only for chat now. Dialing from this is on the roadmap.
